src/tools/toolbox.py

```diff
@@ -32,3 +33,7 @@ def __init__(self):
             raise OSError("Platform not supported!")
         else:
             init_functions[_platform]()
+
+
+def build_filename(media_type: str):
+    return "{}/instantshare_{}_{}.png".format(gettempdir(), media_type, strftime("%Y-%m-%d_%H-%I-%S"))
```

Comment:
Calling `build_filename` with `media_type=text` results in a file named `<tempdir>/instantshare_<media_type>_<timestamp>.PNG` which only makes sense for screenshots.